pkg/kepctl/query_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
oo, 2021

Reply:
Thanks for the review. I think we need to change this line every file, where we have Copyright 2020 written

Reply:
Copyright headers of all files need not be changed. IIRC, whenever we create a new file, we use the year of creation.

Reply:
@palnabarun is correct, just when a new file is created.